CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## v1.7.0 - 2022-10-03
+
+### Changed
+
+- #176 - Remove pytest-redislite in favor of pytest-redis.
+- #174 - Update Dockerfile to install build-essential
+
+### Added
+
+- #174 - Add methods to load data from dictionary and enable tree traversal
+- #168 - Add 'skip' counter to diff.summary()
+- #169/#170 - Add documentation about model processing order
+- #121/#140 - Add and configure renovate
+- #140 - Add renovate configuration validation to the CI
```

Comment:
why #140 is duplicated?

Reply:
maybe because it tackles different topics? (similar to #174)

Reply:
Exactly, I wanted to highlight all the individual things that were done there.